blazar/additions/blazar-manager/hooks/on_before_end.py

```diff
@@ -72,11 +72,23 @@ def render_template(**kwargs):
     return templ.render(**kwargs)
 
 
-def send_email(email_host, to, sender, cc=None, bcc=None, subject=None, body=None):
+def send_email(email_host, email_port, email_ssl, email_user, email_password, to, sender, cc=None, bcc=None, subject=None, body=None):
```

Comment:
I'd request that the new arguments `email_port`, `email_ssl`, `email_user`, `email_password`, default to `None`, so that the new script is backwards compatible if loaded into older blazar container.